dist/js/util.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
```

Comment:
Thats correct we shouldnt be modifying the dist folder files but in the source do we want to have a single JS file say : starter_app.js which include's all these and the dist folder will generate 1 starter_app.min.js which we then include in index.html ..

I see now that our index.html file pulls in : /dist/js/starter_app.min.js and +  /dist/js/draw_visualization.js , instead it will consume only starter_app.min.js, which will include everything that this app needs from the JS code to run.